src/index.html

```diff
@@ -9,6 +9,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="asd"
+        >
+      </a>
+      <nav class="nav__list">
+        <ul class="ul_list_header">
+          <li class="li_list_header">
+            <a href="#" class="is-active link">Apple</a>
```

Comment:
Let's make links more informative
```suggestion
            <a href="#apple" class="is-active link">Apple</a>
```